src/signals/incident-management/containers/IncidentCategoryContainer/components/Add/index.js

```diff
@@ -29,10 +31,23 @@ class Add extends React.Component { // eslint-disable-line react/prefer-stateles
             render={({ invalid }) => (
               <form onSubmit={this.handleSubmit}>
                 <div>
-                  <FieldControlWrapper render={SelectInput} name="sub" display="Categorie" control={this.categoryForm.get('sub')} values={subcategoryList} multiple={false} emptyOptionText="Selecteer..." />
+                  <FieldControlWrapper
+                    render={SelectInput}
+                    name="sub"
+                    display="Categorie"
+                    control={this.categoryForm.get('sub')}
+                    values={subcategoryList}
+                    multiple={false}
+                    emptyOptionText="Selecteer..."
+                  />
 
                   <button className="action primary" type="submit" disabled={invalid}>
```

Comment:
Do we also want to disable the button when (loading === true)?

Reply:
yes i will add that